src/Microsoft.IdentityModel.Tokens/ECDsaAdapter.cs

```diff
@@ -31,13 +31,7 @@ internal ECDsaAdapter()
 #if NET472 || NET6_0_OR_GREATER
             CreateECDsaFunction = CreateECDsaUsingECParams;
 #elif NETSTANDARD2_0
-            // Although NETSTANDARD2_0 specifies that ECParameters are supported, we still need to call SupportsECParameters()
-            // as NET461 is listed as supporting NETSTANDARD2_0, but DOES NOT support ECParameters.
-            // See: https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ecparameters?view=netstandard-2.0
-            if (SupportsECParameters())
-                CreateECDsaFunction = CreateECDsaUsingECParams;
-            else
-                CreateECDsaFunction = CreateECDsaUsingCNGKey;
```

Comment:
not sure if removing this was the right thing to do

Reply:
@jennyf19 I think the safest thing to write is:
```cs
#else
if (SupportsECParameters())
    CreateECDsaFunction = CreateECDsaUsingECParams;
else
    CreateECDsaFunction = CreateECDsaUsingCNGKey;
#endif
```
This is because 462 does not have the class ECParameters.
SupportsECParameters(), probes into the runtime to see if the type is available.

Reply:
`SupportsECParameters()` is not available o n 462

Reply:
This may be correct, essentially leaving it the same, changing 461 -> 462
```cs
#if NET472 || NET6_0_OR_GREATER
            CreateECDsaFunction = CreateECDsaUsingECParams;
#elif NETSTANDARD2_0
            // Although NETSTANDARD2_0 specifies that ECParameters are supported, we still need to call SupportsECParameters()
            // as NET462 is listed as supporting NETSTANDARD2_0, but DOES NOT support ECParameters.
            // See: https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ecparameters?view=netstandard-2.0
            if (SupportsECParameters())
                CreateECDsaFunction = CreateECDsaUsingECParams;
            else
                CreateECDsaFunction = CreateECDsaUsingCNGKey;
#else
            CreateECDsaFunction = CreateECDsaUsingCNGKey;
#endif
```

Reply:
done